packer/scripts/configure_proxy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+if [[ ! -z $PROXY_HOST ]]; then
+    export http_proxy="http://$PROXY_UNAME:$PROXY_PASSWORD@$PROXY_HOST:$PROXY_PORT"
+    echo "Acquire::http::Proxy \"http://$PROXY_UNAME:$PROXY_PASSWORD@$PROXY_HOST:$PROXY_PORT\";" \
```

Comment:
We're relying here on the fact that we know that this script will only run in the docker container.  We should add the as-root function here as well to handle any non-docker non-root cases in the future.